lib/memory.js

```diff
@@ -37,18 +37,32 @@ function containerUsagePercentage(memoryContainerUsage=false){
 	if (!(memoryContainerUsage)) {
 		memoryContainerUsage = containerUsage();
 	}
+	const limit = containerMemoryLimit(memoryContainerUsage);
+	return (memoryContainerUsage / limit) * 100;
+}
+
+/**
+ * Reads metrics from `/sys/fs/cgroup/memory`
+ * @returns {Number} container memory limit (bytes)
+ */
+function containerMemoryLimit(memoryContainerUsage=false){
+	if (!(memoryContainerUsage)) {
+		memoryContainerUsage = containerUsage();
+	}
+
 	let limit = readandFormatMetric('memory/memory.limit_in_bytes');
-	if (typeof(memoryContainerUsage) !== "number" || typeof(limit) !== "number" || (isNaN(memoryContainerUsage)) || (isNaN(limit)) ) {
-		throw Error(`One or more metrics are malformed. containerUsage: ${memoryContainerUsage}, limit: ${limit}`);
+	if (typeof(limit) !== "number" || (isNaN(limit)) ) {
+		throw Error(`Memory limit metric is malformed: limit: ${limit}`);
 	}
 	// if it's set to unlimited, return the total memory of the host
 	if (limit >= UNLIMITED_MEMORY_AMOUNT || limit === 0) {
 		limit = os.totalmem();
 	}
-	return (memoryContainerUsage / limit) * 100;
+	return limit;
 }
 
 module.exports = {
 	containerUsage,
-	containerUsagePercentage
+	containerUsagePercentage,
+    containerMemoryLimit
```

Comment:
tabs in this file

Reply:
good catch. Fixed all formatting in all files.